systemtest/jpsreport_tests/method_A/test4/04_method_A.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+#---------
+# 6 pedestrians created at frame >= 9
+# ids are discontinuous
+# creation times in sources are 1, 5, 10, 15, 25, 30
+#---------
```

Comment:
What's the goal of this test?

Reply:
testing if the arrival times, which are known, are calculated correctly.